askama_shared/src/generator.rs

```diff
@@ -1316,7 +1316,7 @@ impl<'a, S: std::hash::BuildHasher> Generator<'a, S> {
                 "cycle" => match args {
                     [arg] => {
                         if matches!(arg, Expr::Array(arr) if arr.is_empty()) {
-                            panic!("loop.cycle(…) cannot use an empty array.");
+                            return Err("loop.cycle(…) cannot use an empty array.".into());
```

Comment:
This should also not end in a period (I do realize it was there already).

Reply:
I removed the period.